libraries/AP_NavEKF/EKFGSF_yaw.cpp

```diff
@@ -226,8 +226,8 @@ void EKFGSF_yaw::predictAHRS(const uint8_t mdl_idx)
 
     // Gyro bias estimation
     const ftype gyro_bias_limit = radians(5.0f);
-    const ftype spinRate = ang_rate_delayed_raw.length();
-    if (spinRate < 0.175f) {
+    const ftype spinRate_squared = ang_rate_delayed_raw.length_squared();
+    if (spinRate_squared < 0.175f*0.175f) {
```

Comment:
We would need to check the average value of spinrate_squared as we don't want to try compare huge a with the new extremely low value

Reply:
It's not *that* small :-)